### PR TITLE
Improve script usability

### PR DESCRIPTION
**Pull request by kaapstorm**
This PR is in response to some of your feedback, @esoergel. :thinking: I may rename the repo too (because now it really is a typo).

### CODE REVIEW

**Comment by kaapstorm** [comment]
Gonna self-merge, because this is needed. Please feel free to add comments.

File: import_tes.py [hunk 1]
```diff
@@ -80,7 +80,38 @@
 COMMCARE_BASE_URL = 'https://www.commcarehq.org/'
 DHIS2_PAGE_SIZE = 50
 
-# End of configurable constants
+
+# TODO: Confirm `get_name()` uses the right tracked entity attributes
+#       and returns the CommCare case name in the right format:
+def get_name(tracked_entity) -> str:
```
Comment (esoergel) [inline import_tes.py:86]: hah, was about to suggest you move this to the top - beat me to it.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-import_tes
-==========
+Import TEIs
```
Comment (esoergel) [inline README.md:1]: :tada:

**Review by esoergel — COMMENTED** [review]
looks great! no further comments

**Pull request merged**